templates/cluster-template.yaml

```diff
@@ -84,114 +84,74 @@ spec:
   - "${SSH_KEY}"
   tags: []
 ---
-apiVersion: cluster.x-k8s.io/v1alpha3
-kind: Machine
+apiVersion: cluster.x-k8s.io/v1alpha2
```

Comment:
Why is this v1alpha2? Should it not be v1alpha3?

Reply:
typo!